test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -90,7 +94,9 @@ private void assertCommandBehavior(String inputCommand,
         //Confirm the state of data is as expected
         assertEquals(expectedAddressBook, addressBook);
         assertEquals(lastShownList, logic.getLastShownList());
-        assertEquals(addressBook, saveFile.load());
+        if (command.isMutating()) {
```

Comment:
Good job using `isMutating()` to help in testing. This ensures that even with new commands that mutate, they will still be tested.